net.solarnetwork.node.datum.energymeter.mock/src/net/solarnetwork/node/datum/energymeter/mock/MockEnergyMeterDatumSource.java

```diff
@@ -35,8 +35,12 @@
 	private String frequency = "50";
 	private String resistance = "10";
 	private String inductance = "10";
+	private String randomness = "false";
```

Comment:
I just noticed you're using strings for all setting values, but really you should use types appropriate for the properties. The settings code will convert to the data type for you, so your setter methods can be type-specific as well.

For example I'd expect to see

```java
private boolean randomness = false;

public void setRandomness(boolean value) {
    this.randomness = value;
}
```